src/components/earn/PoolCardTri/PoolCardTriRewardText.tsx

```diff
@@ -100,7 +102,7 @@ export default function PoolCardTriRewardText({ apr, inStaging, nonTriAPRs, isLe
   }
 
   // If multiple rewards, render aggregate APR, token logos, and tooltip
-  const totalAPR = (nonTriAPRs ?? []).reduce((acc, { apr: nonTriAPR }) => acc + nonTriAPR, apr)
+  const totalAPR = roundApr((nonTriAPRs ?? []).reduce((acc, { apr: nonTriAPR }) => acc + nonTriAPR, apr))
```

Comment:
This is a little messy.  I'd move the `roundDecimal` to line `113` instead of here, since 105 is logical and 113 is presentational